.github/workflows/continous-integration-docker.yml

```diff
@@ -245,7 +245,7 @@ jobs:
   regenerate-stateful-disks:
     needs: [ build, get-available-disks ]
     uses: ./.github/workflows/deploy-gcp-tests.yml
-    if: ${{ (!fromJSON(needs.get-available-disks.outputs.zebra_checkpoint_disk) && github.event_name == 'push') || github.event.inputs.regenerate-disks == 'true' }}
+    if: ${{ !fromJSON(needs.get-available-disks.outputs.zebra_checkpoint_disk) || github.event.inputs.regenerate-disks == 'true' }}
```

Comment:
I don't know if these changes are due to a merge conflict, or they are actually in this PR.

Can we move them to PR #4385 @gustavovalverde ?